<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditBook</class>
 <widget class="QDialog" name="EditBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>450</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="titleBook">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Название</string>
   </property>
  </widget>
  <widget class="QLabel" name="authorBook">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Автор</string>
   </property>
  </widget>
  <widget class="QLabel" name="releaseBook">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Год выпуска</string>
   </property>
  </widget>
  <widget class="QLabel" name="publishigBook">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Издательство</string>
   </property>
  </widget>
  <widget class="QLabel" name="in_stockBook">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>270</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>В наличии</string>
   </property>
  </widget>
  <widget class="QLabel" name="ratingBook">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>330</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Рейтинг</string>
   </property>
  </widget>
  <widget class="QLabel" name="urlBook">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>390</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Ссылка</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="titleEditBook">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>30</y>
     <width>271</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="authorEditBook">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>90</y>
     <width>271</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="releaseEditBook">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>150</y>
     <width>271</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="publishingEditBook">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>210</y>
     <width>271</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="in_stockEditBook">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>270</y>
     <width>271</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ratingEditBook">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>330</y>
     <width>271</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="urlEditBook">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>390</y>
     <width>271</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditBook</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditBook</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
